### ISSUE TITLE
Angular app without "#" in url can not be accessed

### ISSUE THREAD

**Issue by avozicov** [DevExpress/testcafe#1912]
### Are you requesting a feature or reporting a bug?
- reporting a bug

### What is the current behavior?
- Angular app without "#" in url can not be accessed 

### What is the expected behavior?
- Angular app without "#" in url can be accessed 

### How would you reproduce the current behavior (if this is a bug)?
- Try to access the following url with your test script:  https://manage-test.adtechmedia.io/accounts/signup 

#### Provide the test code and the tested page URL (if applicable)
Tested page URL:
- https://manage-test.adtechmedia.io/accounts/signup 
Test code
```
import { Selector } from 'testcafe';

fixture `Angular TodoMVC`
  .page('https://manage-test.adtechmedia.io/accounts/signup');
 
test('Check the application title', async t => {
  const appTitle = Selector('h1');

  await t.expect(appTitle.textContent).eql('todos');
}); 
```
### Specify your

* operating system: macOS Sierra version 10.12.6
* testcafe version: 0.18.1
* node.js version: v8.5.0

**Comment by AlexanderMoskovkin** [comment]
Hi @avozicov,

Thanks for your report. I've reproduced the issue.

If I set the 'https://manage-test.adtechmedia.io/' url as a test page it works correctly. But If I set the signup page ('https://manage-test.adtechmedia.io/accounts/signup') as a test page it's broken.
We'll investigate it and will notify you about our progress.

I've reproduced it in `hammerhead-playground` as well. /cc @miherlosev @LavrovArtem

**Comment by avozicov** [comment]
Hi @AlexanderMoskovkin ,
Thank you for the prompt reply.

**Comment by miherlosev** [comment]
Hi @avozicov,
 
I see the problem in our [proxy](http://github.com/devexpress/testcafe-hammerhead).
I've created a separate issue regarding it: https://github.com/DevExpress/testcafe-hammerhead/issues/1349

**Comment by AlexanderMoskovkin** [comment]
The fix is published in `testcafe@0.18.4`

**Comment by avozicov** [comment]
@AlexanderMoskovkin @miherlosev @VasilyStrelyaev 
Great! Thank you very much!
